packages/core-components/src/components/MarkdownContent/MarkdownContent.tsx

```diff
@@ -64,10 +64,18 @@ const useStyles = makeStyles<BackstageTheme>(
   { name: 'BackstageMarkdownContent' },
 );
 
+type TransformLink = (href: string) => string;
+
 type Props = {
   content: string;
   dialect?: 'gfm' | 'common-mark';
-  linkTarget?: Options['linkTarget'];
+  linkTarget?: React.HTMLAttributeAnchorTarget | TransformLink;
+  transformLinkUri?: TransformLink;
+  transformImageUri?: (
```

Comment:
Let's have both of these use just `(href: string) => string` inlined, not feeling it's worth the extra named type tbh, as we'd need to export that.